src/dotnet/APIView/APIViewWeb/Languages/SwaggerLanguageService.cs

```diff
@@ -5,7 +5,6 @@
 using System.IO;
 using System.Threading.Tasks;
 using ApiView;
-using APIViewWeb.Helpers;
 using Microsoft.ApplicationInsights;
 
 namespace APIViewWeb
```

Comment:
Line 23. Swagger is still `UsesTreeStyleParser  = false`. For consistency, we should change this too

Reply:
Swagger is not using tree token yet.

Reply:
hahaha i completely forgot :( my bad :(